examples/interconnects.py

```diff
@@ -63,15 +73,32 @@
 # Get the first two Interconnects
 print("\nGet the first two interconnects")
 try:
-    interconnects = oneview_client.interconnects.get_all(0, 2)
+    interconnects = interconnect.get_all(0, 2)
     pprint(interconnects)
 except HPOneViewException as e:
     print(e.msg)
 
+# Turn the power off
+print("\nTurn the power off and the UID light to 'Off' for interconnect ")
+try:
+    interconnect = interconnect.patch(
+        operation='replace',
+        path='/uidState',
+        value='On'
```

Comment:
The Comment says "power off", while the value for the rest call is 'On'. 
Also for the operation mentioned in the print statement, we need to work on /powerState and not /uidstate. Can you please check once.